### PR TITLE
New parser for /etc/ceph/ceph.conf

### PR DESCRIPTION
**Pull request by psachin**
Signed-off-by: Sachin Patil <psachin@redhat.com>

### CODE REVIEW

**Comment by xiangce** [comment]
Seems there is some sensitive information in this file, you might need to filter it. please create a JIRA request at first.

**Comment by psachin** [comment]
JIRA: https://projects.engineering.redhat.com/browse/INSPEC-110

File: insights/parsers/ceph_conf.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    >>> conf.get('client.radosgw.gateway', 'user') == 'apache'
+    True
+    >>> conf.get('client.radosgw.gateway', 'log_file') == '/var/log/ceph/radosgw.log'
+    True
+"""
+from .. import IniConfigFile, parser, add_filter
+from insights.specs import Specs
+
+add_filter(Specs.ceph_conf, ["[",
+                             "rgw_swift_account_in_url"])
```
Comment (lhuett) [inline insights/parsers/ceph_conf.py]: Rules should add filters for specific INI items such as "rgw_swift_account_in_url", this should not be in the parser.

Comment (xiangce) [reply insights/parsers/ceph_conf.py]: @psachin - Please remove this filter and add it to the plugin.

**Pull request merged**